src/defaultsettings.cpp

```diff
@@ -69,6 +69,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("enable_build_where_you_stand", "false" );
 	settings->setDefault("3d_mode", "none");
 	settings->setDefault("3d_paralax_strength", "0.025");
+	settings->setDefault("3d_convergence_distance", "1.95");
```

Comment:
Since `3d_paralax_strength` is not explained there I (wrongly) assumed that not all settings defined here are to be explained there. Fixed now